package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.1",
     "algoliasearch": "^4.8.3",
+    "classnames": "^2.3.1",
```

Comment:
@sherakama I know we went with `cnbuilder` for Online Giving and SAA Homesite for chaining classnames. Would you rather we go with that (I think you shared another newer tool option recently too) or `classnames` that @mattanglin has installed here?
cnbuilder:
https://github.com/xobotyi/cnbuilder

Reply:
I'm not terribly opinionated on classname packages since they're just simple string aggregators. I only chose `classnames` since it was already installed as a sub-dependency of some other packages. 🤷🏼 
![image](https://user-images.githubusercontent.com/1952337/129991268-be8f14e7-4936-4f5b-914b-30edcc0fe6be.png)

One of my goals is to start integrating [bundle-analyzer](https://www.gatsbyjs.com/plugins/gatsby-plugin-webpack-bundle-analyser-v2/) so we can get a more holistic view of what's in our bundles.

Reply:
Woot. Looking forward to your work with bundle-analyzer

Reply:
Yeah +1. This would be cool!